packages/web-worker/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export {expose} from './worker';
+export {createWorker} from './parent';
```

Comment:
Is `parent` the right term to use here. In an application, the UI thread spawns a worker thread so the application thread would be the parent. Unless this is a different context that is. 

Maybe just call it `create-worker.ts`?

Reply:
Hmm, I feel like it's appropriate, because `createWorker` is for the application, which is the "parent" in the relationship. In the end it doesn't really matter much IMO because this is just so that they are in separate files for tree shaking.